internal/pkg/diff/reporter.go

```diff
@@ -130,7 +182,10 @@ func pathToString(path cmp.Path) string {
 		case cmp.MapIndex:
 			parts = append(parts, cast.ToString(v.Key().Interface()))
 		case cmp.SliceIndex:
-			parts = append(parts, cast.ToString(v.Key()))
+			// index1 or index2 can be "-1",
+			// if the value is on one side only
+			index1, index2 := v.SplitKeys()
+			parts = append(parts, cast.ToString(math.Max(float64(index1), float64(index2))))
```

Comment:
A este jedna dobnost, ... ak sa porovnavaju 2 `slices`, a v jednej polozka je a v druhej nie, ... tak to vo vypise vracalo ako index `-1` ... upravene aby to vratilo ten index, ktory existuje.